src/main/java/com/asyncy/apis/hub/controllers/UserController.java

```diff
@@ -15,4 +20,17 @@
     public User get(final HttpServletRequest request) {
         return fetchUser(request);
     }
+
+    @RequestMapping(path="/user", method = DELETE)
+    public void invalidate(final HttpServletRequest request, final HttpServletResponse response) {
+        UserCreds owner = fetchOwner(request);
+        if (owner.getUuid() != null && !owner.getUuid().equals("")) {
```

Comment:
Handy one liner to do the same.

```suggestion
        if (!"".equals(owner.getUuid())) {
```

Reply:
Btw, even more handier is to use StringUtils.isNotBlank() from the Apache commons lang library. https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNotBlank-java.lang.CharSequence-

Reply:
Indeed